Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM ruby:3.0.5
```

Comment:
I've added a checklist item to the Ruby 3.3 PR to upgrade this dockerfile after merging